src/rp2_common/pico_stdio/include/pico/stdio.h

```diff
@@ -49,6 +49,9 @@ typedef struct stdio_driver stdio_driver_t;
  * Call this method once you have set up your clocks to enable the stdio support for UART, USB
  * and semihosting based on the presence of the respective libraries in the binary.
  *
+ * When stdio_usb is configured, this method can be optionally made to block, waiting for a connection
+ * via the variables specified in \ref stdio_usb_init (i.e. \ref PICO_STDIO_USB_POST_CONNECT_WAIT_DELAY_MS)
```

Comment:
I wonder if this should mention `PICO_STDIO_USB_CONNECT_WAIT_TIMEOUT_MS` instead, because AFAICT `PICO_STDIO_USB_POST_CONNECT_WAIT_DELAY_MS` has no effect if `PICO_STDIO_USB_CONNECT_WAIT_TIMEOUT_MS` is left at its default value of 0 ?

Reply:
copy/paste error again